lib/db/schema/patch-067-068.sql

```diff
@@ -0,0 +1,155 @@
+ALTER TABLE `unverifiedTokens`
+ADD COLUMN `shortCodeHash` BINARY(32) DEFAULT NULL,
+ALGORITHM = INPLACE, LOCK = NONE;
+
+CREATE INDEX `unverifiedTokens_shortCodeHash`
+ON `unverifiedTokens` (`shortCodeHash`)
+ALGORITHM = INPLACE LOCK=NONE;
```

Comment:
I wonder if it's worth making this a unique index on `(shortCodeHash, uid)` in order to prevent the accidental creation of duplicate shortcodes for different login sessions by the same user.  That should never happen in theory, but sometimes it's worth ensuring that theory and practice will match :-)